docs/tutorials/simple-date-clock-application.adoc

```diff
@@ -0,0 +1,482 @@
+
+CloudHarness allows you to quickly setup app that needs to be ran in a kubernete cluster.
+It generates the initial files and folders for your project depending on some templates tacking different aspects of your app depending on your requirements, __e.g.__, for a webapp project, it generates the frontend initial files for ReactJS and the initial Flask files for the backend.
+For the API part, CloudHarness relies on OpenAPI 3 to deal with the endpoints/model description.
+
+The different aspects that will be covered here are:
+
+* how to install CloudHarness locally on your machine;
+* how to quickly setup a first version of a local cluster using `minikube`;
+* how to build and locally deploy the projects from the repository;
+* how to bootstrap a new app, build it and run it;
+* how to modify/update the app, built it and run it again.
+
+== The tools you need to deploy/build your application
+
+The following tools, beside python, are not required to work with CloudHarness.
+They are here to deal with the local Kubernete cluster creation, monitoring and the application deployment.
+Before installing everything, please be sure you have the following tools installed on your machine:
+
+* `python`
+* `minikube` -- the local mini cluster
+* `kubectl` -- to help controlling the cluster
+* `helm` -- to deal with Kubernete "packages" (helm is basically a package manager for Kubernete)
+* `skaffold` -- to build/deploy/run the apps
+
+
+== Install CloudHarness (if it is not yet done)
+
+CloudHarness is coded in Python, consequently, it's always better to create a local virtualenv dedicated to the project to avoid messing with your system dependencies.
+As shown in the snippet above, we will clone the repository, change directory inside the freshly clone directory and create a virtualenv inside (for this tutorial, the Python's version used is CPython 3.10.6).
+
+First step is to clone the https://github.com/MetaCell/cloud-harness[CloudHarness repository] on your system.
+
+
+.Cloning and installing CloudHarness
+[source,bash]
+----
+git clone git@github.com:MetaCell/cloud-harness.git
+cd cloud-harness
+python -m venv --symlinks venv     # this creates a virtualenv in the "venv" directory
+----
+
+Now that the virtualenv is created, we need to activate it to work in isolation.
+
+.Activating the virtualenv and installing CloudHarness
+[source,bash]
+----
+source venv/bin/activate
+sh ./install.sh
+----
+
+To test if your installation went successfully, check if at least the following command exist: `harness-application`.
+
+.Checking the installation
+[source,bash]
+----
+harness-application --help
+----
+
+If everything is good, you're in the right path to use CloudHarness!
+
+
+== How to setup your local cluster using minikube (if not done yet)
+
+CloudHarness is designed to help you generate all the required artifacts for a deployment on a Kebernete cluster.
```

Comment:
Small typo: Kebernete -> Kubernetes

Reply:
Noted :). I'll add a variable in the document for nouns (cloudharness and kubernetes), I know myself and I'll do the mistake other times otherwise.